libdrgn/configure.ac

```diff
@@ -37,6 +37,21 @@ AS_CASE(["x$enable_openmp"],
 AC_SUBST(OPENMP_CFLAGS)
 AC_SUBST(OPENMP_LIBS)
 
+AC_ARG_ENABLE([asan],
+	      [AS_HELP_STRING([--enable-asan@<:@=ARG@:>@],
+			      [use ASAN. ARG may be yes or no.
+			       @<:@default=no@:>@])],
+	      [], [enable_asan=no])
+
+SANITIZER_CFLAGS=
+SANITIZER_LDFLAGS=
+AS_CASE(["x$enable_asan"],
+	[xyes], [SANITIZER_CFLAGS="$SANITIZER_CFLAGS -fsanitize=address -fno-omit-frame-pointer"
+			 SANITIZER_LDFLAGS="$SANITIZER_LDFLAGS -fsanitize=address"],
+	[xno], [])
```

Comment:
The convention for simple yes/no options is to do something like `AS_IF([test "x$enable_asan" != xno], [...])`.